### ISSUE TITLE
toggling fullscreen mode 1 on a floating window on and off will not revert it to its original geometry

### ISSUE THREAD

**Issue by Daxtorim** [hyprwm/Hyprland#1233]
### Steps to reproduce 
(start with a tiled window)
```sh
hyprctl dispatch togglefloating ""
hyprctl dispatch resizeactive exact 600 200
hyprctl dispatch centerwindow ""
sleep 1
hyprctl dispatch fullscreen 1
sleep 1
hyprctl dispatch fullscreen 1
hyprctl dispatch moveactive 100 100
```

### Expected outcome
The floating window should be returned to its original size.

### Actual outcome
Window keeps the the fullscreen size but can be moved around.

### Notes
The issue does not occur with `fullscreen 0`.

PS.:
Why do dispatchers *require* at least one argument. Kinda annoying to always put `""` after dispatchers that don't take arguments.

**Comment by vaxerski** [comment]
> Why do dispatchers require at least one argument. Kinda annoying to always put "" after dispatchers that don't take arguments.

They don't, but hyprctl wont accept no arg

**Comment by vaxerski** [comment]
fixed in 2076905d6efe05cc0f79995d73a18c3b8661ce21

**Comment by Daxtorim** [comment]
Thank you! :+1: 

There is still one small issue:
With `dwindle:no_gaps_when_only=true` set, after toggling `fullscreen 1` the floating window does not have a border or rounded corners anymore.

Top left has the blue "active window" border, the other two tiled windows the gray "inactive window" border, the floating window is missing any border after the fullscreen toggle.
![image](https://user-images.githubusercontent.com/72616153/208312612-23f125e3-f33f-411a-ad0c-f3b3798ebc4f.png)


(Does not happen with `fullscreen 0`, although that won't bring the border back if it was lost prior)
